modules/alloydb/README.md

```diff
@@ -0,0 +1,183 @@
+# AlloyDB module
+
+This module manages the creation of an AlloyDB cluster and instance with potential read replicas and an advanced cross region replication setup for disaster recovery scenarios. 
+It can also create an initial set of users via the `users` parameters.
+
+Note that this module assumes that some options are the same for both the primary instance and the secondary one in case of cross regional replication configuration.
+
+*Warning:* if you use the `users` field, you terraform state will contain each user's password in plain text.
+
+<!-- TOC -->
+* [AlloyDB module](#alloydb-module)
+  * [Examples](#examples)
+    * [Simple example](#simple-example)
+    * [Cross region replication](#cross-region-replication)
+  * [Variables](#variables)
+  * [Outputs](#outputs)
+<!-- TOC -->
+
+## Examples
+### Simple example
+
+This example shows how to setup a project, VPC and AlloyDB cluster and instance.
+
+```hcl
+module "project" {
+  source          = "./fabric/modules/project"
+  billing_account = var.billing_account_id
+  parent          = var.folder_id
+  name            = "alloydb-prj"
+  prefix          = var.prefix
+  services = [
+    "servicenetworking.googleapis.com",
+    "alloydb.googleapis.com",
+  ]
+}
+
+module "vpc" {
+  source     = "./fabric/modules/net-vpc"
+  project_id = module.project.project_id
+  name       = "my-network"
+  # need only one - psa_config or subnets_psc
+  psa_configs = [{
+    ranges          = { alloydb = "10.60.0.0/16" }
+    deletion_policy = "ABANDON"
+  }]
+  subnets_psc = [
+    {
+      ip_cidr_range = "10.0.3.0/24"
+      name          = "psc"
+      region        = var.region
+    }
+  ]
```

Comment:
nit: inconsistent style. Pick one of `[{` or `[\n{`

Reply:
Updated!